examples/tls/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  client:
+    build:
+      context: .
+    command: bash -c "sh /app/test.sh"
+    links:
+      - triton-server
+    volumes:
+      - .:/app
+      - ./certs:/certs
+  triton-redis:
+    image: redis
+    command: "redis-server /conf/redis.conf"
+    ports:
+      - '6379:6379'
+    volumes:
+      - ./certs:/certs
+      - ./conf:/conf
+  triton-server:
+    links:
+      -   triton-redis
+    build:
+      context: ../..

```

Comment:
Please also update the root server Dockerfile and any other NGC image version locatiosn to use 23.06.